CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## CHANGELOG FOR 1.1.x
 #### 1.1.0
+- bugfix [#1502](https://github.com/ergonode/backend/issues/1502) Error on filtering by numeric attribute in adding relation grid (wiewiurdp)
```

Comment:
The bug was not yet released therefore there's no need for changelog entry not issue